### DISCUSSION TITLE
zh/guide/drivers/yandex

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#60] [Announcements]
# zh/guide/drivers/yandex

AList v3的文档

https://alist.nn.ci/zh/guide/drivers/yandex.html

**Comment by ji1360677347** [comment]
为什么百度网盘用了几天好好的，视频突然就播放不了了，其它的文档和音乐还可以正常打开，试过重新添加或者换设备也不行了，求大神解答！！！！！！

**Comment by ChetHub** [comment]
被yandex拒绝。
You are now leaving Yandex
It is forbidden to receive tokens with those rights for this application (unauthorized_client)
You will now be redirected to the client page.
Please note: Yandex has no affiliation to this page.

**Comment by simplove** [comment]
必须科学上网才能下载，国内用没什么意义
